### PR TITLE
Docs(dev-setup) small changes to instructions

### PR DESCRIPTION
**Pull request by fafcrumb**
Switched to dual-booting ubuntu and used my instructions to set things up. It revealed a couple small flaws.

### CODE REVIEW

**Comment by robertavram** [comment]
when you have a chance.. please check this again.. Zoli added some stuff to this file as well,

Update from develop when ready

File: dev_environ_setup_linux.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ $ sudo -u postgres psql postgres
 Step 3. Set Postgres user password to "postgres", and create PostGIS required extensions:
 
 ```bash
-# alter user postgres with password 'postgres';
```
Comment (robertavram) [inline dev_environ_setup_linux.md]: I thought we need to use the "WITH" word in the statement. check postgres docs

Comment (fafcrumb) [inline dev_environ_setup_linux.md]: That's correct. Typo

Comment (fafcrumb) [inline dev_environ_setup_linux.md]: Fixed

**Pull request merged**